### ISSUE TITLE
Compile error "unresolved import `hyper::http::Incoming`. There is no `Incoming` in `hyper::http`" with v0.1.1

### ISSUE THREAD

**Issue by mputz86** [GGist/ssdp-rs#11]
Hey there,
with a current Rust release I am not able to get the library build.
```
   Compiling ssdp v0.1.1 (https://github.com/GGist/ssdp-rs#e4b21ef5)
   /home/mputz/.cargo/git/checkouts/ssdp-rs-49f0d9b2719c6b74/master/src/message/ssdp.rs:9:25: 9:33 error: unresolved import `hyper::http::Incoming`. There is no `Incoming` in `hyper::http` [E0432]
```
Since I am rather new to Rust, I am not totally sure if it is not my mistake. I think my setup is pulling a "too new" version of the hyper crate. But since ssdp-rs is not restricting the version, this seems to be valid and therefore brakes.

Many thanks in advance!

**Comment by GGist** [comment]
Hey thanks for opening this issue.

You are exactly correct, I hadn't updated this crate for quite some time (back when cargo still used github to pull in dependencies) and it looks like the new build system doesn't pull in Cargo.lock when pulling from crates.io. That being said, I have pinned the major versions in Cargo.toml and updated the library to use the newest dependencies, including hyper.

The minor version for this crate has been bumped to 0.1.2 so make sure you use that. I tested on Windows and Ubuntu using Rust Nightly 1.4 and can see messages getting transmitted and received in the examples, let me know if this fixes it on your end.
